scripts/archive_api.py

```diff
@@ -512,14 +708,87 @@ async def stream_message_list(archiveClient, db_records, next_offset):
 	yield b"\x00\x00" # NUL terminators for the array and the overall document
 	
 
-@app.get("/topic/{topic_name}")
+@app.get("/topic/{topic_name}",
+         description="Retrieve messages which were published on a given topic during a specified time range. "
+                     "Time ranges are specified as Kafka times (milliseconds since the unix epoch). " #TODO: is this always UTC?
+                     "Results are returned in chunks, so users should expect to repeat requests to "
+                     "obtain all messages in a time range. "
+                     "A response with an empty list of messages indicates a lack of further messages "
+                     "(i.e. after the requested offset) in the specified range. "
+                     "Responses will contain messages in time-order. "
+                     "Authentication via SCRAM is required to fetch non-public messages.",
```

Comment:
This is a get request, so I assume the time ranges are specified in url params? If so can they be documented here?